grub-core/lib/gpt.c

```diff
@@ -425,13 +425,21 @@ grub_gpt_read_backup (grub_disk_t disk, grub_gpt_t gpt)
   grub_disk_addr_t addr;
 
   /* Assumes gpt->log_sector_size == disk->log_sector_size  */
-  if (grub_gpt_disk_size_valid(disk))
+  if (gpt->status & GRUB_GPT_PRIMARY_HEADER_VALID)
+    {
+      sector = grub_le_to_cpu64 (gpt->primary.alternate_lba);
```

Comment:
It looks like this file isn't consistent with use of whitespace between a function name and it's args - are there different circumstances when one or the other is used? (If not, we may want to pick one for our patches and stick with it)